roles/content_view_version_cleanup/tasks/find_unused_cv_versions.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: "find content view versions of {{ cv.name }}"
+  theforeman.foreman.resource_info:
+    server_url: "{{ server_url | default(omit) }}"
+    username: "{{ username | default(omit) }}"
+    password: "{{ password | default(omit) }}"
+    validate_certs: "{{ validate_certs | default(omit) }}"
+    organization: "{{ organization }}"
+    resource: content_view_versions
+    params:
+      content_view_id: "{{ cv.id }}"
+  register: versions
+
+- name: "delete unused content view versions of {{ cv.name }}"
```

Comment:
file is called "find_", so when I see such a task included, I would not think it will delete anything

Reply:
I thought about that too… `find_and_delete_unused_…` sounds so long? But probably still more correct.